Makefile

```diff
@@ -8,6 +9,10 @@ all: test
 build:
 	go build
 
+.PHONY: generate
+generate:
```

Comment:
why phony, should we just have `bindata.go` as the target, and have it depend on the `templates/*`?

Reply:
Oh good call.

Reply:
I'm going to keep a phony generate but make it depend on bindata.go.